core-blocks/archives/index.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Server-side rendering of the `core/archives` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/archives` block on server.
+ *
+ * @see WP_Widget_Archives
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the post content with archives added.
+ */
+function render_block_core_archives( $attributes ) {
+	static $block_id = 0;
+	$block_id++;
```

Comment:
The way this is used in tandem with `ServerSideRender`, this will generally lead to ID conflicts. From my own local testing:

```
document.querySelectorAll( '#wp-block-archives-1' )
```

returns multiple elements if I have more than one Archive block in my editor.

Reply:
Worth noting that implementing an `edit` for this block type would allow the editor to retain more control over this kind of information.

Reply:
Added using `uniqid` instead, thoughts?

Reply:
@mcsf Also disabled the pointer-events of the links, this way it'll not cause the leaving page alert, however, the links are not clickable within the editor. Probably better but not ideal, thoughts?

Reply:
> Added using `uniqid`

That should do for now, thanks.

> disabled the pointer-events of the links 

It doesn't solve for other interactions (e.g. when attribute _Display as dropdown_ is on). It's indeed not ideal, but I don't have a good solution. I mean, sure, we could also disable pointer-events on `select` elements, but then even more interaction is lost, perhaps leading the user to perceive the experience as "buggy" and, thus, confusing. On the other hand, hijacking `select` elements to remove their `onchange` attribute could technically help, but that sounds like a very bad precedent to set. /cc @mtias